WPMedia/arc-themes-components
# Update format social url

## Ticket

- [TMEDIA-725](https://arcpublishing.atlassian.net/browse/TMEDIA-725)

## Description

Small update to the formatSocialURL method to detect a preformatted url and just return it (Facebook/RSS)

## Acceptance Criteria

N/A

## Test Steps

1. Checkout branch - `git checkout update-format-social-url`
2. Run Tests `npm run test`
3. Verify the tests work, check my logic.

## Author Checklist


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
